com.spotify.Client.json

```diff
@@ -130,7 +130,8 @@
                         "rmdir usr",
                         "mkdir -p export/share/applications",
                         "sed s/Icon=spotify-client/Icon=com.spotify.Client/ share/spotify/spotify.desktop > export/share/applications/com.spotify.Client.desktop",
-                        "echo StartupWMClass=Spotify >> export/share/applications/com.spotify.Client.desktop",
+                        "sed -i s/StartupWMClass=spotify/StartupWMClass=Spotify/ export/share/applications/com.spotify.Client.desktop",
```

Comment:
Use `desktop-file-edit`

Reply:
At what point do we just say forget editing their .dektop file and just provide our own?

Reply:
It would make localization possible.

Reply:
Just look at the list of places Spotify is available:
 https://support.spotify.com/us/using_spotify/the_basics/full-list-of-territories-where-spotify-is-available/

As of now though everything is in English.

Reply:
It might also be nice to make the appstream stuff translatable too.

Reply:
Nobody is going to translate the file...